cluster-bootstrap/src/main/scala/akka/management/cluster/bootstrap/ClusterBootstrap.scala

```diff
@@ -64,6 +64,26 @@ final class ClusterBootstrap(implicit system: ExtendedActorSystem) extends Exten
 
   private[this] val _selfContactPointUri: Promise[Uri] = Promise()
 
+
+  private val autostart = system.settings.config.getStringList("akka.extensions")
+    .indexOf(classOf[ClusterBootstrap].getName) > -1
```

Comment:
A little bit hackish to look at the `akka.extension` list, but there's no public API way I can think of to identify when its loaded through config.

Reply:
It may look a little less hackish if we use `contains`, no?